### PR TITLE
Update spec graph

### PR DESCRIPTION
**Pull request by FintanH**
* Remove `refs` portion under `$TRACKED_PEER_ID`
* Add `rad/refs`

### CODE REVIEW

File: docs/rfc/identity_resolution.md [hunk 1]
```diff
@@ -81,20 +81,21 @@ The namespacing scheme could look as follows:
                 |-- heads # <-- code branches owned by $PEER_ID go here
                 |-- rad
                 |   |-- id # <-- points to the identity document history
+                |   |-- refs # <-- signature document
```
Comment (kim) [inline docs/rfc/identity_resolution.md:84]: Nope, not this one -- its the blob which contains the signed refs of the peer. Feel free to conjure a different name if this sounds like `malkovich, malkovich`. `signed_refs` for example.

File: docs/rfc/identity_resolution.md [hunk 2]
```diff
@@ -81,20 +81,21 @@ The namespacing scheme could look as follows:
...
-                            `-- rad
-                                |-- id
-                                |-- self
-                                `-- ids
-                                    |-- $CERTIFIER[1]
-                                    `-- $CERTIFIER[2]
+                        |-- heads
+                        `-- rad
+                            |-- id
+                            |-- refs
```
Comment (kim) [inline docs/rfc/identity_resolution.md:94]: Dito

File: docs/rfc/identity_resolution.md (hunk 1 above)
Comment (FintanH) [reply docs/rfc/identity_resolution.md:84]: Wait... The jokes has me confused :sweat_smile: It _is_ called `refs` at the moment right? Are you just commenting on the description or the name or both?

**Comment by kim** [comment]
> It is called refs at the moment right? Are you just commenting on the description or the name or both?

It is called refs at the moment, yes. The code refers to it as
`rad_refs`. As this thing got you confused several times, perhaps now
is a good time to name it differently, so it conveys better what its
purpose is.

**Pull request merged**